tensorflow/lite/micro/kernels/logistic.cc

```diff
@@ -27,11 +28,59 @@ namespace tflite {
 namespace ops {
 namespace micro {
 namespace activations {
-
+namespace {
```

Comment:
Why do we need to make this anonymous namespace?

Reply:
This utility function is used in this specific file only. I've seen it done in many other reference kernels (see for example depthwise_conv.cc where OpData and CalculateOpData are defined in an anonymous namespace)

Reply:
Thank you. This makes sense now.